epam/async-race.md

```diff
@@ -186,7 +186,13 @@ Also, please deploy the UI to GitHub Pages, Netlify, Vercel, Cloudflare Pages, o
 ## 🎨 Prettier and ESLint Configuration (10 points)
 
 - [ ] **Prettier Setup (5 points):** Prettier is correctly set up with two scripts in `package.json`: `format` for auto-formatting and `ci:format` for checking issues.
-- [ ] **ESLint Configuration (5 points):** ESLint is configured with the [Airbnb style guide](https://www.npmjs.com/package/eslint-config-airbnb). A `lint` script in `package.json` runs ESLint checks. Configuration files should reflect strict TypeScript settings as per `tsconfig.json`.
+- [ ] **ESLint Configuration (5 points):** ESLint is configured using the recommended rulesets:
+  - `eslint:recommended`
+  - `plugin:@typescript-eslint/recommended`
+  - `plugin:react/recommended`
+  - `plugin:react-hooks/recommended`  
```

Comment:
To me, the listed rule format looks like a deprecated one. ESLint 9 now uses a new "flat config" system